.gitignore

```diff
@@ -38,3 +38,13 @@ yarn-error.log
 cypress/screenshots/
 cypress/videos/
 cypress/downloads/
+
+# yarn
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
+yarn-error.log*
+yarn-debug.log*
```

Comment:
You have a missing newline here, you can add `  "files.insertFinalNewline": true,` to your vscode settings.